lib/glimesh_web/templates/oauth2_provider/application/show.html.eex

```diff
@@ -0,0 +1,59 @@
+<div class="container" style="text-align:center;">
+  <h1><%= gettext("Show Application") %></h1>
+
+  <ul class="list-group">
+    <li class="list-group-item">
+      <strong><%= gettext("Name:") %></strong>
+      <%= @application.name %>
+    </li>
+    <li class="list-group-item">
+      <strong><%= gettext("ID:") %></strong>
+      <%= @application.uid %>
+    </li>
+    <li class="list-group-item">
+      <button class="btn btn-dark" onclick="toClipboard()">
+        <%= gettext("Copy Secret") %>
+      </button>
+
+      <div style="position:relative; overflow:hidden;">
+        <input style="position:absolute; top:-100px;" id="secretKey" value="<%= @application.secret %>"/>
```

Comment:
More css stuff

Reply:
this css should in realistically not be needed anywhere else as this is to just hide the secret from the user to use it with the copy button